site/pages/type-of-consequence.tsx

```diff
@@ -0,0 +1,112 @@
+import { NextPageContext } from "next";
```

Comment:
This file will just need renaming to `type-of-consequence.page.tsx`

Reply:
done